config/app.php

```diff
@@ -100,7 +100,8 @@
     |--------------------------------------------------------------------------
     |
     | This key is used by the Illuminate encrypter service and should be set
-//
+
+'App\Plugins\Stripe\ServiceProvider',
 'App\Plugins\stripe\ServiceProvider',
 'App\Plugins\stripr\ServiceProvider',//
 'App\Plugins\Paypal\ServiceProvider',//
```

Comment:
this is not the correct place to register ServiceProviders

Reply:
Those were unnecessary few lines added. Removed